src/registrar/templates/domain_request_intro.html

```diff
@@ -14,6 +14,11 @@ <h1>You’re about to start your .gov domain request.</h1>
         <h2>Time to complete the form</h2>
         <p>If you have <a href="{% public_site_url 'domains/before/#information-you%E2%80%99ll-need-to-complete-the-domain-request-form' %}" target="_blank" class="usa-link">all the information you need</a>,
            completing your domain request might take around 15 minutes.</p>
+       {% if has_profile_feature_flag %}
+       <h2>How we'll reach you</h2>
+       <p>While reviewing your domain request, we may need to reach out with questions. We'll also email you when we complete our review If the contact information below is not correct, visit <a href="" target="_blank" class="usa-link">your profile</a> to make updates.</p>
```

Comment:
Blocking: Missing href value

Reply:
This will be implemented in a future ticket